package.json

```diff
@@ -1,15 +1,16 @@
 {
   "name": "@magnet.me/logger-js",
-  "version": "1.0.3",
+  "version": "1.0.4",
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
-  "types": "dist/index.d.ts",
+  "types": "dist/esm/index.d.ts",
   "repository": "https://github.com/Magnetme/logger-js",
   "author": "Magnet.me",
   "license": "MIT",
   "private": false,
   "scripts": {
     "build": "tsc -p tsconfig.json && tsc -p tsconfig-cjs.json",
+    "clean": "rm -rf dist && yarn run build",
```

Comment:
Sorry I wasn't clear: with `clean` meant just the `rm -rf dist` command. Right now it's not clear this is also build. Either just do the clean command, or a `cleanbuild` with these two combined.

Reply:
I've given the user the option to do both now. yarn clean if you just want to remove the dist folder and cleanbuild for if you're working on something.